Samples/tutorials/first-run-experience-tutorial/README.md

```diff
@@ -0,0 +1,106 @@
+---
+page_type: sample
+urlFragment: office-add-in-first-run-experience-tutorial
+products:
+  - office
+languages:
+  - javascript
+extensions:
+  contentType: samples
+  technologies:
+    - Add-ins
+  createdDate: 09/05/2024 4:00:00 PM
+description: "A completed version of the step-by-step first-run experience tutorial hosted on learn.microsoft.com."
+---
+
+# Office Add-in First-run Experience Tutorial - Completed
+
+## Summary
+
+This sample is the result of completing the [Build an Office Add-in with a basic first-run experience](https://learn.microsoft.com/office/dev/add-ins/tutorials/first-run-experience-tutorial). It was constructed with the [Yeoman generator for Office Add-ins](https://learn.microsoft.com/office/dev/add-ins/develop/yeoman-generator-overview).
+
+The tutorial gives step-by-step instructions on how to add functionality alongside explanations as to why code is being added. Use this sample if you want to explore and try the completed code, or if you need to debug any issues you encountered while following the tutorial.
+
+## Features
+
+This sample demonstrates the basics of implementing a first-run experience (FRE) in Excel. The pattern can be applied to other Office applications where Office Web Add-ins are supported.
+
+## Applies to
+
+- Excel on Windows
+- Excel on Mac
+- Excel on the web
```

Comment:
Maybe this is a limit of the readme format, but having it only "applying to" Excel limits the cross-application functionality that's mentioned earlier.